psi4/share/cmake/psi4/psi4Config.cmake.in

```diff
@@ -74,7 +74,7 @@ set(psi4_FOUND TRUE)
 set(PSI4_FOUND TRUE)
 
 function(add_psi4_plugin TARGET SOURCES)
-    # remove ${TARGET} from ${ARGV} to use ${ARGV} as ${SOURCES}
```

Comment:
Yeah, those too. I am just thinking In general if the config file can define as many relevant things as possible it would make things easier on the plugin developer. That assumes a plugin in question is fairly feature rich and being developed as a production level package like forte, for example. 

On the other hand, if the plugin is created by a student to get familiar with some of psi4's innards, [like TDC's programming project #7](http://sirius.chem.vt.edu/wiki/doku.php?id=crawdad:programming:project7) (which is a little out of date now) then it could be useful for the student to see those flags defined in their `CMakeLists.txt` rather than have it be a very simple few lines with a lot of magic done by the `add_psi4_plugin` function.